### DISCUSSION TITLE
Contact form validation

### DISCUSSION THREAD

**Discussion by rneo** [bootscore/bootscore#641] [Q&A]
Nie wiem co jest nie tak, ale u mnie cały czas na zielono oznacza pola nawet gdy występują błędy: https://mdk-soft.com.pl/kontakt/

I don't know what's wrong, but for me the fields are marked in green all the time even when there are errors: https://mdk-soft.com.pl/kontakt/

**Comment by crftwrk** [comment]
I cannot check this because your site is password protected. Please remove password protection.

**Reply by crftwrk** [reply]
Class `wpcf7-not-valid` is missing in the form.

![screenshot](https://github.com/bootscore/bootscore/assets/51531217/59b3a49f-1b9f-446a-b80d-dc3dfb95d4e6)

This class overrides the success validation. I do not see a JS error. Are you on latest CF7 version and are sure that you have setup mandatory fields with * `[text* first-name]`?

I would also try to quickly remove the password protection in htaccess file to test, nobody knows.

**Reply by rneo** [reply]
Jest to najnowszy CF7, wraz z skopiowaną wersją od Was i troszkę zmienioną, nie wiem czemu nie ma tych class
This is the latest CF7, with a version copied from you and slightly changed, I don't know why there are no these classes

**Reply by crftwrk** [reply]
The `wpcf7-not-valid` class comes from CF7, not from our plugin. As mentioned above, do try and error and remove htaccess password protection from your site. Maybe CF7 doesn't like it.
